frontend/pages/projects.tsx

```diff
@@ -23,7 +25,15 @@ const Projects: NextPage = () => {
                     }
                 )
                     .then((response) => response.json())
-                    .catch((error) => console.log(error));
+                    .catch((err) => {
+                        if (notify) {
+                            notify(
+                                "Something went wrong:" + err,
+                                NotificationType.ERROR,
+                                2000
+                            );
+                        }
+                    });
```

Comment:
In my experience the `err` is either `undefined` or a very convoluted message. So it would not include it in the error message and say something like `Sorry, we couldn't load the projects.`

Reply:
the err is the error message that we get from the server

Reply:
Backend errors usually are short and concise

Reply:
Not in the catch. The catch is when you cannot reach the server or something goes wrong client side.

Backend errors are when `response.success == false` and are stored in `response.reason`

Reply:
@Mouwrice You were right I changed the popups